### PR TITLE
This commit also adds edified fences as well as the sign textures 

### PR DESCRIPTION
**Pull request by Falkory220**

### CODE REVIEW

File: Common/src/main/java/at/petrak/hexcasting/datagen/recipe/HexplatRecipes.java
Comment (object-Object) [inline Common/src/main/java/at/petrak/hexcasting/datagen/recipe/HexplatRecipes.java]: why does this give 3 now instead of 4?

File: Common/src/main/java/at/petrak/hexcasting/common/blocks/decoration/BlockHexFenceGate.java
Comment (SamsTheNerd) [inline Common/src/main/java/at/petrak/hexcasting/common/blocks/decoration/BlockHexFenceGate.java:16]:
```java
public BlockHexFenceGate(Properties $$0) {
    super($$0, WoodType.DARK_OAK);
}
```
should probably give this a more descriptive name than $$0. I mean i guess it doesn't super matter since it's a very generic constructor but still.

Same thing on BlockHexFence.

File: Fabric/src/main/resources/fabric.mod.json
Comment (SamsTheNerd) [inline Fabric/src/main/resources/fabric.mod.json:1]: is the `"paucal": "*"` right ? i assume it was changed to fix something but i would think it should have some more specific value ?

**Review by SamsTheNerd — CHANGES_REQUESTED** [review]
looks good to me except for maybe those two comments I left.

I'm not the most familiar with hex's datagen and whatnot though so I can't say for sure that it's not missing something somewhere but it looks thorough to me !

**Comment by object-Object** [comment]
let's just wait for a response from Cypher on the Blanketcon thing before we merge this.

**Review by object-Object — APPROVED** [review]
send it

**Pull request merged**
